.changeset/added-static-list.md

```diff
@@ -0,0 +1,16 @@
+---
+"@salt-ds/lab": minor
+---
+
+Added `StaticList`, `StaticListItem` component to lab.
+
+```tsx
+<StaticList>
+  <StaticListItem>
+    <Text>New static list feature updates are available in lab</Text>
+  </StaticListItem>
+  <StaticListItem>
+    <Text>New static list item feature updates are available</Text>
```

Comment:
nit/fyi.. if this is intentionally different that’s fine but otherwise the text is different

Reply:
it was added intentionally, but it does not makes sense so made both of them same.